src/components/N4CFeatures.js

```diff
@@ -57,6 +57,11 @@ const features = {
             svg: "/img/nfdi4chem_Medicinal-Pharmaceutical_Chemistry.svg",
             link: "/docs/pharmaceutical_chemistry",
         },
+        {
```

Comment:
The domains could be sorted in alphabetical order. This will shift the synthesis domain to the end, but I think it is less confusing.